File: server/card_server/Card/db_dir/queries.py
class Queries:
    def __init__(self):
        pass

    def select_all(self):
        return 'SELECT * FROM card_db;'

    def select_one(self):
        return 'SELECT * FROM card_db WHERE card_id = %s;'

    def insert_card(self):
        return 'INSERT INTO card_db (account_id, inside, mid, three, passing, steal, block, imgurl, createdat, updatedat) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, NOW(), NOW()) RETURNING *;'
    
    def delete_card(self):
        return 'DELETE FROM card_db WHERE card_id = %s RETURNING *;'
    

File: server/src/ProposedTrades/db/queries.ts
import { DbQueryType } from '../../db/dbTypes'

export const getAllQuery = (): DbQueryType => {
    return {
        query: "SELECT * FROM proposed_trade_db;"
    }
}

export const getOneQuery = (): DbQueryType => {
    return {
        query: "SELECT * FROM proposed_trade_db WHERE id = ($1);"
    }
}

export const getDeleteQuery = (): DbQueryType => {
    return {
        // query: "DELETE * FROM proposed_trade_db WHERE id = ($1) RETURNING *;"
        query: "DELETE FROM proposed_trade_db WHERE id = ($1) RETURNING *;"
    }
}

export const insertQuery = (): DbQueryType => {
    return {
        query: "INSERT INTO proposed_trade_db (from_id, to_id, from_cards, to_cards, createdat, updatedat) VALUES ($1, $2, $3, $4, NOW(), NOW()) RETURNING *;"
    }
}

